app/cloud/index.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Kong Cloud
+layout: default
+hide_subscribe: true
+id: cloud
+---
+
+<section class="section enterprise-section">
+  <div class="container">
+    <div class="row">
+      <div class="one-half column">
+        <h2>The Power of Kong, now delivered SaaS</h2>
+        <p>Kong Cloud comes with high-performance hosting, SLA, Backups, HA, 24x7 Support and more.</p>
+        <p>
+          <a href="#request-demo" class="button button-primary button-large scroll-to">Request Demo</a>
+        </p>
+      </div>
+      <div class="one-half column">
+        <figure>
+          <img src="/assets/images/enterprise/enterprise.svg" width="440" height="222" alt="Enterprise"/>
+        </figure>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section demo-request-section" id="request-demo">
+  <div class="container">
+    <header class="section-header">
+      <h2>START YOUR KONG CLOUD</h2>
+      <p>You're Just a Few Clicks Away</p>
+    </header>
+
+    <form class="demo-request-form">
+      <div class="spinner">
+        <div class="bounce1"></div>
+        <div class="bounce2"></div>
+        <div class="bounce3"></div>
+      </div>
+
+      <div class="alert alert-success" role="alert"><center>Thank you! We'll be in touch soon.</center></div>
+
+      <div class="input-group">
+
+        <label for="company-name">Company Name</label>
+        <input id="company-name" name="company" type="text" placeholder="Mashape Inc" required>
+
+        <label for="full-name">Full Name</label>
+        <input id="full-name" name="name" type="text" placeholder="Jason Businessman" required>
+
+        <label for="title">Title</label>
+        <input id="title" name="title" type="text" placeholder="CEO">
+
+        <label for="work-email">Work Email</label>
+        <input id="work-email" name="email" type="email" placeholder="jason@mashape.com" required>
+
+        <label for="phone">Phone <small>(optional)</small></label>
+        <input id="phone" name="phone" type="text" placeholder="415 000 0000">
+
+        <label for="tell-us-more">Tell us more</label>
```

Comment:
I removed the checklist inputs for 'On Prem deployment' and 'Cloud Deployment'. Are they necessary, since it's a cloud product?